lib/common.js

```diff
@@ -43,16 +43,18 @@ function cleanApp (app) {
 }
 
 // TODO add an optional parse function
-const doRequest = (url, headers) => new Promise(function (resolve, reject) {
-  debug('Making request: %s %j', url, headers);
+const doRequest = ({ url, headers, requestOptions }) => new Promise(function (resolve, reject) {
```

Comment:
why does this take a single object parameter instead of three parameters?

Reply:
I think this syntrax is better to pass in params optionally. I can change it to take 3 arguments instead of an object if necessary.

Reply:
mm I guess it's ok to take an object argument, but I think in that case:
1- method should be a top level propery, so the most basic fields (method, url, headers) are handled outside the options.
2- the options property should be named `options` here, since it's a request function, it's redundant the options apply to the request. (the public property in the scraper methods should still be named requestOptions though)